src/test/java/com/nbu/project/LogisticsCompanyApplicationTests.java

```diff
@@ -1,13 +0,0 @@
-package com.nbu.project;
-
-import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
-
-@SpringBootTest
-class LogisticsCompanyApplicationTests {
-
-	@Test
-	void contextLoads() {
```

Comment:
Why delete the default test?

Reply:
This is an integration test that fails without running the database and I think we don't want that kind of tests in the project